_layouts/blog.html

```diff
@@ -0,0 +1,108 @@
+---
+layout: internal
+---
+<section class="row solid-post">
+    <section class="solid-breadcrumbs col-xs-12">
```

Comment:
The comment reg section as a child of section applies here as well. Maybe change `<section class="solid-breadcrumbs col-xs-12">` to `<nav aria-label="breadcrumbs" class="solid-breadcrumbs col-xs-12">`